ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/ResourceMapperImpl.java

```diff
@@ -72,9 +80,10 @@ public class ResourceMapperImpl implements ResourceMapper {
         this.resources = resources;
         this.additionalData = additionalData;
         this.treatNilableAsOptional = treatNilableAsOptional;
+        this.hateoasMapper = new HateoasMapper();
     }
 
-    public void setOperation() {
+    public void setOperation(String packageId, SemanticModel semanticModel, ServiceDeclarationNode service) {
```

Comment:
Why do we need packageId for here?

Reply:
Fixed with [63ac241d79644776e53dc2f338009c91970f6df7](https://github.com/ballerina-platform/openapi-tools/pull/1589/commits/63ac241d79644776e53dc2f338009c91970f6df7)